### ISSUE TITLE
Write §3.1 Configuration-time errors

### ISSUE THREAD

**Issue by tfpauly** [taps-api/drafts#15]
Explain how to process application preferences should be treated when they have conflicts, etc. Probably error out immediately to avoid creating impossible connections.

**Comment by theri** [comment]
Hi, I'm currently working on an initial version of the pre-establishment configuration in the API, see #2. 
I would be willing to contribute an initial version of this Section, which I think is related, e.g., how do we prioritize application requirements and preferences, what do we communicate back and when...

**Comment by britram** [comment]
Sold!

**Comment by tfpauly** [comment]
Sounds perfect!

**Comment by theri** [comment]
Pushed a first version. Comments welcome.
I'm assuming "Configuration-time" is supposed to mean Pre-Establishment, as it is under that heading, even though (some) configuration can also happen later.
Thinking about different errors, I also included some errors that the transport system can only know about later. 
Note that we try to eliminate possible errors of "Setting a property at the wrong time" (e.g., Protocol Selection Property after Establishment) in the API.
